core/modules/user/css/user.css

```
@@ -108,6 +108,7 @@
 .password-strength.weak .indicator {
   background-color: #ff0000;
 }
+.password-strength.tooweak .password-strength-text,
```

```suggestion
.password-strength.too-weak .password-strength-text,
```

Same reasons why we don't have the other class be `.passwordstrength`